manifests/y/Youdao/YoudaoDict/9.1.6.0/Youdao.YoudaoDict.locale.zh-CN.yaml

```diff
@@ -0,0 +1,36 @@
+# Created with YamlCreate.ps1 v2.2.0 using InputObject 🤖 $debug=QUSU.CRLF..7-2-7
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Youdao.YoudaoDict
+PackageVersion: 9.1.6.0
+PackageLocale: zh-CN
+Publisher: 网易公司
+PublisherUrl: https://cidian.youdao.com/
+# PublisherSupportUrl:
+PrivacyUrl: https://c.youdao.com/dict/law/youdao_agreement/privacy.html
+Author: Youdao
+PackageName: 网易有道词典
+# PackageUrl:
+License: Proprietary
+# LicenseUrl:
+# Copyright:
+# CopyrightUrl:
+ShortDescription: 网易有道词典 9  ——比强大更强大的桌面翻译软件
+# Description:
+Moniker: youdaodict
+Tags:
+- cloud
+- dict
+- dictionary
+- online
+# Agreements:
+ReleaseNotes: 
+- 文本翻译支持应用个人术语库
+- 文档翻译新增小语种：中文德文互译、中文俄文互译、中文法文互译、中文葡萄牙文互译、中文西班牙文互译
+- 文档翻译支持上传xlsx格式文件并导出译文
```

Comment:
```suggestion
ReleaseNotes: |-
  - 文本翻译支持应用个人术语库
  - 文档翻译新增小语种：中文德文互译、中文俄文互译、中文法文互译、中文葡萄牙文互译、中文西班牙文互译
  - 文档翻译支持上传xlsx格式文件并导出译文
```

Reply:
I'm not a YAML expert, but these additions allow Winget to install this package in my VM.